src/core/Akka.Remote/Endpoint.cs

```diff
@@ -240,12 +240,13 @@ internal sealed class InvalidAssociation : EndpointException, IAssociationProble
         /// <summary>
         /// TBD
         /// </summary>
+        /// <param name="message">TBD</param>
         /// <param name="localAddress">TBD</param>
         /// <param name="remoteAddress">TbD</param>
         /// <param name="cause">TBD</param>
         /// <param name="disassociateInfo">TBD</param>
-        public InvalidAssociation(Address localAddress, Address remoteAddress, Exception cause = null, DisassociateInfo? disassociateInfo = null)
-            : base(string.Format("Invalid address: {0}", remoteAddress), cause)
+        public InvalidAssociation(string message, Address localAddress, Address remoteAddress, Exception cause = null, DisassociateInfo? disassociateInfo = null)
```

Comment:
Could you please add these as constructor overloads, rather than modifying the original constructors? I doubt that anyone is using these public APIs in their own code but for the sake of consistency, let's not modify the public API if we can avoid it.